.eslintignore

```diff
@@ -0,0 +1 @@
+test
```

Comment:
eslint ignoring should only be done in confit files via ignorePatterns, and we absolutely must lint tests.